src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -92,20 +93,34 @@ export default {
      * Create a report comment notification
      *
      * @param {Object} params
+     * @param {Object} params.report
      * @param {Object} params.reportAction
      * @param {Function} params.onClick
      * @param {Boolean} usesIcon true if notification uses right circular icon
      */
-    pushReportCommentNotification({reportAction, onClick}, usesIcon = false) {
+    pushReportCommentNotification({report, reportAction, onClick}, usesIcon = false) {
         const {person, message} = reportAction;
         const plainTextPerson = _.map(person, (f) => f.text).join();
+        const isChatRoom = ReportUtils.isChatRoom(report);
 
         // Specifically target the comment part of the message
         const plainTextMessage = (_.find(message, (f) => f.type === 'COMMENT') || {}).text;
 
+        let title = '';
+        let body = '';
```

Comment:
```suggestion
        let title;
        let body;
```

NAB. No need to set a init value here.